src/main/resources/avro/readmethods.avdl

```diff
@@ -12,6 +12,11 @@ This request maps to the body of `POST /reads/search` as JSON.
 If a reference is specified, all queried `ReadGroup`s must be aligned
 to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
+
+_Error Conditions_
+
+- If `readGroupIds` is empty, this method will throw `GAException` and set the HTTP
```

Comment:
Definitely a 400. I agree this documentation should live at a more general level, we have many places in the schemas where we say a parameter is "required". If these parameters are not provided, the server should return a 400.

I can take a stab at adding a section about this somewhere, but I'm not sure where it should live in the documentation branch.